lib/membrane/rtcp/app.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Membrane.RTCP.App do
```

Comment:
It's a bit ambiguous, we call application modules like that. I would make it `AppPacket`

Reply:
What's behind this `Specific` suffix? IMHO, the best way would be to append `Packet` to everything but for Report, but even having ReportPacket is clearer than having `.App` or `.Bye`

Reply:
Or maybe RTCP.Packet**s**.App

Reply:
Specific comes from the RFC: `APP:  Application-specific functions`. It could also be `Defined` suffix: `6.7 APP: Application-Defined RTCP Packet` Anyway, I'll add Packet suffix, after all, these modules won't be used often outside the parser

Reply:
+ lone Packet might be a bit confusing